public/omni/omni-cluster-setup/registering-machines/join-machines-to-omni.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Join machines to Omni
+---
+
+Omni lets you connect existing Talos nodes, whether they’re running on bare metal, in QEMU, or on cloud instances, to your Omni environment so it can automatically take over their management.
+
+You can do this in two ways:
+
+* **Machine Join Config**: Apply a configuration file to a Talos node to register it with Omni.
+
+* **Kernel Parameters**: Add kernel parameters to your machine’s boot configuration so it automatically connects to Omni during startup.
+
+> **Note:** Once a machine joins Omni, it is managed entirely through Omni.
+> The local Talos API is disabled for security reasons, and all future configuration changes must be made via Omni.
+
+The following sections explain each method in more detail and how to use them.
+
+## Machine Join Config
+
+The Machine Join Config is a lightweight configuration file that lets you  register existing Talos nodes with Omni.
+Use this approach for nodes you created independently, for example, with `talosctl cluster create`, Proxmox, or cloud VMs.
+
+These nodes must be in **maintenance mode**, meaning no machine configuration (`controlplane.yaml` or `worker.yaml`) has been applied yet.
+
+### How to use the Machine Join Config file
+
+Follow these steps to download and apply the Machine Join Config file:
+
+1. Log in to your Omni dashboard.
+
+2. Select **Download Machine Join Config** to download the configuration file.
+
+![Download Machine Join Config Button](./images/join-machine-machine-join-config.png)
+
+3. Apply the configuration to your Talos nodes by running:
```

Comment:
I would rather remove it, primary way should be to supply this join config as userdata on your specific platform: nocloud for Proxmox, userdata on AWS/GCP/..., etc.

This apply-config would break when the machine is removed.